src/component/project/Project/CreateProject/CreateProject.tsx

```diff
@@ -33,8 +36,10 @@ const CreateProject = () => {
         e.preventDefault();
         clearErrors();
         const validName = validateName();
+        const validProjectId = validateProjectId();
```

Comment:
Do we need another local validation the id when we are already asking the API for a validation?